src/client.zig

```diff
@@ -120,11 +131,19 @@ pub const Client = struct {
         user_data: u128,
         callback: Request.Callback,
         operation: Operation,
-        data: []const u8,
+        message: *Message,
+        body_length: usize,
```

Comment:
Let's rename `body_length` to `body_size` in keeping with `message_size_max`, `@sizeOf`, `usize` etc.